packages/side-nav/src/CollapsedSideNavItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Portal from '@leafygreen-ui/portal';
+import { cx, css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { useSideNavContext } from './SideNavContext';
+
+const collapsedItemStyles = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  border-bottom: 1px solid ${uiColors.gray.light2};
+  color: ${uiColors.green.dark2};
+
+  &:first-of-type {
+    border-top: 1px solid ${uiColors.gray.light2};
+  }
+`;
+
+const collapsedItemActiveStyle = css`
+  background-color: ${uiColors.green.light3};
+`;
+
+interface CollapsedSideNavItemProps {
+  // React Node rendered when the navigation is collapsed.
```

Comment:
are we commenting rather than ts-docing here intentionally?

Reply:
For some reason I was thinking tsdoc was compatible with both single-line and multi-line JS comments. Fixed.